pkg/asset/config/appliance_config.go

```diff
@@ -214,6 +247,22 @@ func (a *ApplianceConfig) validateConfig() field.ErrorList {
 	return allErrs
 }
 
+func (a *ApplianceConfig) validateImageRegistry() field.ErrorList {
+	if a.Config.ImageRegistry == nil {
+		return nil
+	}
+
+	if a.Config.ImageRegistry.Port != nil {
```

Comment:
Maybe add some TODO on validating also the URI

Reply:
will do

Reply:
Done